docs/pages/architecture.md

```diff
@@ -4,37 +4,50 @@ title: Meshery Architecture
+<h4>The Meshery architecture can be observed in two perspectives:</h4> 
+ 
+##### 1. <a href="#clients"><b>Clients</b></a>
+##### 2. <a href="#providers"><b>Providers</b></a> 
 
-## Clients
+<br /><a href="{{site.baseurl}}/assets/img/architecture/meshery-architecture-diagram.svg"><img src="{{site.baseurl}}/assets/img/architecture/Meshery-architecture-diagram.png" /></a>
 
-<a href="{{site.baseurl}}/assets/img/architecture/meshery-client-architecture.svg"><img src="{{site.baseurl}}/assets/img/architecture/meshery-client-architecture.svg" /></a>
 
-## Providers
+<a name="clients">
 
-<a href="{{site.baseurl}}/assets/img/architecture/meshery-provider-architecture.svg"><img src="{{site.baseurl}}/assets/img/architecture/meshery-provider-architecture.svg" /></a>
+### 1. <b>Clients</b>
 
-## Network Ports
+<a href="{{site.baseurl}}/assets/img/architecture/meshery-client-architecture.svg"><img src="{{site.baseurl}}/assets/img/architecture/Meshery-client-architecture.svg" /></a>
+
+<a name="providers">
+
+### 2. <b>Providers</b>
+
+<a href="{{site.baseurl}}/assets/img/architecture/meshery-provider-architecture.svg"><img src="{{site.baseurl}}/assets/img/architecture/Meshery-provider-architecture.svg" /></a>
+
+#### <b>Network Ports</b>
 
 Meshery uses the following list of network ports to interface with its various components:
 
-| Adapter                                        | Port             |
-| :--------------------------------------------- | :--------------- |
-| Meshery REST API                               | 9081/tcp      
-| Learn Layer5 Application                       | 10011 |
-| {% assign adaptersSortedByPort = site.adapters | sort: 'port' -%} |
+| Network Application                            | Port             |
+| :--------------------------------------------- | :--------------: |
+| Meshery REST API                               | 9081/tcp         |
+| Learn Layer5 Application                       | 10011            |
+
+#### <b>Adapter Ports</b>
 
-{% for adapter in adaptersSortedByPort -%}
+| Service Mesh  | Port          |
+| :------------ | ------------: |
+{% for adapter in site.adapters -%}
 {% if adapter.port -%}
-| [{{ adapter.name }}]({{ site.baseurl }}{{ adapter.url }}) | {{ adapter.port }} |
+| <img src="{{ adapter.image }}" style="width:20px" /> [{{ adapter.name }}]({{ site.baseurl }}{{ adapter.url }}) |&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; {{ adapter.port }} |
 {% endif -%}
 {% endfor %}
 
-See the [Adapters](/docs/architecture/adapters) section for more information on the function of an adapter.
+See the [**Adapters**](/docs/architecture/adapters) section for more information on the function of an adapter.
 
-## Statefulness in Meshery components
+#### <b>Statefulness in Meshery components</b>
```

Comment:
Let's pull the styling out and avoid inline style like this.

Reply:
Not the end of the world, if this is just the occasional need. Ideally, we'll avoid intermixing html and markdown in the same line. 

My comments might be pre-optimizations that we can address later. Keep the docs coming, @chandrashritii